WordPress/Classes/ViewRelated/Stats/Shared Views/Date Chooser/SiteStatsTableHeaderView.swift

```diff
@@ -139,6 +143,37 @@ private extension SiteStatsTableHeaderView {
     }
 
     func displayDate() -> String? {
+        guard let components = displayDateComponents() else {
+            return nil
+        }
+
+        let (from, to) = components
```

Comment:
This is a bit nitpicky, but can we give these `from` and `to` vars more descriptive names? Like maybe `fromDate` and `toDate` or something like that?